src/main/ApplicationImpl.cpp

```diff
@@ -554,7 +554,8 @@ ApplicationImpl::getJsonInfo(bool verbose)
 void
 ApplicationImpl::reportInfo(bool verbose)
 {
-    mLedgerManager->loadLastKnownLedger(/* restoreBucketlist */ false);
+    static_cast<LedgerManagerImpl&>(*mLedgerManager)
```

Comment:
why not add the method to LedgerManager? Casting to an impl class seems like an anti-pattern we generally avoid in prod paths

Reply:
Well, these are not quite 'prod' paths, are they? This version of LCL loading is rather hacky and shouldn't be used most of the time, outside of the few utils. Which is why I wasn't sure about making it a part of the 'public' LedgerManager interface. I don't have too strong opinion on this, just wanted to make this option a bit harder to use on accident. But maybe it's in overkill, I'm not really sure.